File: c# fundamentals/exercises/Data Types and Methods/Data Types and Methods/VowelOrDigit/VowelOrDigit.cs
using System;

namespace VowelOrDigit
{
    class VowelOrDigit
    {
        static void Main(string[] args)
        {
            char current = char.Parse(Console.ReadLine());
            if (current == 'a' || current == 'e' || current == 'o' || current == 'i' || current == 'u')
            {
                Console.WriteLine("vowel");
            } 
            else if (current <= '9' && current >= '0')
            {
                Console.WriteLine("digit");
            }
            else
            {
                Console.WriteLine("other");
            }
        }
    }
}

File: TechModule/Java/src/vowelOrDigit.java
import java.util.Scanner;

/**
 * Created by Alec on 28-Jul-16.
 */
public class vowelOrDigit {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String character = scan.next();
        char ch = character.charAt(0);
        if (Character.isDigit(ch)) {
            System.out.println("digit");
        } else if (Character.isAlphabetic(ch)) {
            System.out.println("vowel");
        } else {
            System.out.println("other");
        }
    }
}
